openbsd/Battery.c

```diff
@@ -1,74 +0,0 @@
-/*
-htop - openbsd/Battery.c
-(C) 2015 Hisham H. Muhammad
-(C) 2015 Michael McConville
```

Comment:
Attribution lost after moving the code.

Reply:
openbsd/Platform.c already has these authors copyright annotations listed.  Likewise for the other cases you mentioned.